### ISSUE TITLE
Mention KB terms of service in frontend

### ISSUE THREAD

**Issue by jgonggrijp** [UUDigitalHumanitieslab/I-analyzer#555]
Quote @JosedeKruif 

> De KB heeft de gebruikscontracten en voorwaarden vernieuwd. In de nieuwe voorwaarden staat dat er nu ook een vermelding binnen I-analyzer moet komen te staan. Ik citeer:
>> duidelijk kenbare gebruiksvoorwaarden op I-Analyzer te vermelden, waarin staat: ‘De informatie op I-Analyzer die afkomstig is van Delpher/DBNL [wat van toepassing is] mag u alleen gebruiken voor niet-commercieel onderzoek. U mag deze informatie niet delen (zoals via besloten netwerken of internet) of doorleveren aan andere personen of partijen. Voor andersoortig gebruik dient u contact op te nemen met de KB via Dataservices@KB.nl.

**Comment by lukavdplas** [comment]
We support adding a terms of service page to the corpus definition now. I guess this just applies to the `dutchnewspapers-all` corpus?

**Comment by jgonggrijp** [comment]
José's quote seems to suggest that these terms also apply to the dbnl corpus. I don't know the history of that corpus, though, so I can't really tell with certainty.

**Comment by lukavdplas** [comment]
Yes, I'm not sure why that quote mentions it. The DBNL corpus was added later, in 2023. It's based on the DBNL dataset which is publicly available, so we did not contact the KB at the time. Our lab has done some other DBNL-related projects in the past, though, so perhaps it refers to that? Or the text is just boilerplate since the KB manages both datasets.

In any case, the DBNL corpus in I-analyzer only includes data in the public domain. The [terms of service for the DBNL dataset](https://www.kb.nl/onderzoeken-vinden/datasets/dbnl-dataset) state:

> Gebruik van deze dataset valt, op basis van het auteursrecht, uiteen in 2 mogelijkheden. Werken waarvan de maker langer dan 70 jaar geleden overleden is, vallen in het publieke domein. Je mag ze dus vrij gebruiken. Een groot deel van deze werken is als [zip-bestand](https://www.dbnl.org/letterkunde/pd/index.php) beschikbaar. Andere werken zijn auteursrechtelijk beschermd en op aanvraag beschikbaar voor onderzoeksdoeleinden, bijvoorbeeld door wetenschappers, onderzoekers, docenten of journalisten. Vraag hiervoor toegang via [dataservices@kb.nl](mailto:dataservices@kb.nl).

So everything in the linked zip file (which is what I downloaded for I-analyzer) is unrestricted.

Delpher has a similar deal, so that's why I thought the terms of service were probably only relevant for the complete version, not the public one.

**Comment by lukavdplas** [comment]
Discussed this with Jill, who is satisfied with the update in #1776. Users who access the full corpus already sign a contract that specifies the terms of service, and those contracts are updated every once in a while, so providing a copy on the interface is a) redundant, and b) could result in inconsistencies.
